### ISSUE TITLE
PDK should support all keys in metadata.json

### ISSUE THREAD

**Issue by rmestrum** [puppetlabs/pdk-planning#28]
Now only for a few keys in metadata.json default values can be provided. For example: `name`, `author`, `license` 

For other keys, such as `project_page`,`summary`, can't be provided with defaults

Request is to add support for all keys of the metadata.json so they can be provided with default values.

**Comment by scotje** [comment]
This is specifically in regards to the items tracked in the `~/.pdk/cache/answers.json` file?

**Comment by rmestrum** [comment]
The file `~/.pdk/cache/answers.json` is a possibility. It would be nice if it is possible to add these to the template.

**Comment by scotje** [comment]
It seems like `project_page`, `summary`, etc. are likely to vary for each module, so persistent defaults don't seem to make as much sense here? We could add the ability to set things using command line options to be passed to `pdk new module` though, for example:

```
pdk new module --metadata-project-page="https://github.com/example/project"
```

Would that address your use case?

**Comment by rmestrum** [comment]
Yes that would address my case. With that possibility we can script the creation of a module.

**Comment by scotje** [comment]
Thank you for your feedback! I have created https://tickets.puppetlabs.com/browse/PDK-1385 to track this request in JIRA.
